_posts/2016-12-14-configurarkieserver.md

```diff
@@ -0,0 +1,240 @@
+---
+
+layout: post
+
+title: Como configurar el kie server y usar BRMS como un servicio de reglas.
+
+date: 2016-12-14 13:06
+
+author: Nahuel Persia
+
+categories: kieserver BRMS reglas rest
+
+---
+
+#Como configurar el kie server y usar BRMS como un servicio de reglas.
+
+Cuando queremos aprender sobre las herramientas de la familia de automatización de Red Hat, en general empezamos con el curso de Open.
+Este engloba en forma muy general BPMS y BRMS, con lo cual, se nos empieza a generar una nebulosa de donde termina uno y empieza el otro.
+
+##Requisitos para seguir este post
+* Tener instalado JBoss BRMS 6.x sobre JBoss EAP 6.x o 7.x.
+* Tener conocimientos básicos de JBoss EAP.
+* Asegurar que se encuentran deployados en JBoss EAP los siguientes paquetes: business-central.war, kie-server.war.
+* Tener mínimos conocimientos sobre autoría de reglas de negocio sobre JBoss BRMS 6.x.
+
+La motivación de este post no es enseñar a escribir reglas, ni a usar BRMS, eso lo aprendimos anteriormente. Lo que pretendemos es poder usar las reglas que creamos desde alguna pieza de software externo al motor de reglas, en resumen: **Armar un servicio de reglas**.
+
+Para lograr esto, lo que vamos a hacer es configurar el kie server de BRMS, quien es el encargado de gestionar las comunicaciones entre el motor de reglas y el exterior.
```

Comment:
KIE es una sigla ("Knowledge Is Everything") con lo cual tiene que estar **siempre** en mayúscula.